addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Helios Binding
+Bundle-SymbolicName: org.openhab.binding.helios;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/cxf-core-3.1.0-SNAPSHOT.jar,
+ lib/cxf-services-wsn-api-3.1.0-SNAPSHOT.jar,
+ lib/cxf-services-wsn-core-3.1.0-SNAPSHOT.jar
+Import-Package: com.google.common.collect,
+ com.google.gson,
+ javax.ws.rs.client,
+ javax.ws.rs.core,
```

Comment:
Could you please also add
```
javax.xml.bind
javax.xml.bind.annotation
```
as those packages are not part of the "core" Java libs and some OSGi frameworks thus require the explicit import?